docs/governance.rst

```diff
@@ -0,0 +1,253 @@
+roles where there are conflicts.
+
+If there are concerns with a tedana steering committee member or leader, any
+enforcer of the code of conduct can ask anyone to step down from a leadership role.
+If a person refuses to step down, then an enforcer of the code of conduct can call
+a vote of contributors to remove an individual from a leadership role in tedana.
+
+
+Decision Making Process
+-----------------------
+
+These rules outlined below are inspired by the 
+`decision-making rules for the BIDS standard <https://github.com/bids-standard/bids-specification/blob/master/DECISION-MAKING.md>`_, which in turn were inspired by the
+`lazy consensus system used in the Apache Foundation <https://www.apache.org/foundation/voting.html>`_,
+and heavily depend on the
+`GitHub Pull Request review system <https://help.github.com/articles/about-pull-requests/>`_.
+
+Definitions
+```````````
+
+Repository
+  `https://github.com/ME-ICA/tedana <https://github.com/ME-ICA/tedana>`_
+
+Contributor
+  Person listed in the `all-contributors file`_.
+  The community decides on the content of this file using the same process as any
+  other change to the Repository (see below) allowing the meaning of "Contributor"
+  to evolve independently of the Decision-making rules.
+
+Maintainer
+  A Contributor responsible for the long term health of the project and the
+  community. Maintainers have additional rights (see Rules) helping them to
+  resolve conflicts and increase the pace of the development when necessary.
+  Any maintainer can self-remove themselves. Any contributor can become a
+  maintainer by request and with the support of the majority of the current
+  maintainers. Current Maintainers:
+
+  +-----------------------------------+-----------------+
+  | Name                              | Time commitment |
+  +===================================+=================+
+  | `Logan Dowdle`_ (@dowdlelt)       | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Elizabeth DuPre`_ (@emdupre)     | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Dan Handwerker`_ (@handwerkerd)  | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Ross Markello`_ (@rmarkello)     | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Taylor Salo`_ (@tsalo)           | 3h/week         |
+  +-----------------------------------+-----------------+
+  | `Joshua Teves`_ (@jbteves)        | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Eneko Uruñuela`_ (@eurunuela)    | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Kirstie Whitaker`_ (@KirstieJane)| 0.5h/week       |
+  +-----------------------------------+-----------------+
+
+
+Rules
+`````
+
+1. Potential modifications to the Repository should first be proposed via an 
+   Issue. Rules regarding Votes apply to both Pull Requests and Issues.
```

Comment:
Bit confusing, since we've barely introduced voting yet. Could we cross-ref it, at least ? Or did I miss an earlier introduction of the role of voting ?